releases/cloud/point-2024-11-14-release-note.md

```diff
@@ -0,0 +1,55 @@
+---
+description: This article lists improvements and bug fixes in the Syskit Point Cloud version 2024.6.70.28
+---
+
+# November 14, 2024
+
+[Start a free trial](https://www.syskit.com/products/point/free-trial/) and [tell us what you think](https://www.syskit.com/company/contact-us/).
+
+## About Syskit Point Cloud
+
+* **New to Syskit Point Cloud?** Explore Syskit Point Cloud with a 21-day [free trial](https://www.syskit.com/products/point/free-trial/) for an easy and effective way to manage and secure your environment.
+
+* **Already using Syskit Point Cloud?** Syskit Point Cloud is automatically upgraded to the latest version when available. The automatic update occurs outside working hours to ensure minimal interference with your day-to-day business. The new version will start rolling out with this announcement and is expected to reach all customers in the upcoming days.
+
+## New Features
+
+
+* **New policy available**: Inactive Workspaces
+  * Inactive Workspaces are workspaces that have been inactive for the time period set by administrators. 
+  * This policy helps you clean up inactive workspaces to save storage space and keep your environment running smoothly. 
+    * [For more details about the Inactive Workspaces policy, read this article.](../../governance-and-automation/automated-workflows/inactive-workspaces-admin.md)
+  * Once an inactive workspace is detected, it is flagged as a vulnerability and can be found in the Security & Compliance dashboard.
+    * [To find out more on how to resolve the detected vulnerability, read the related Security & Compliance report article.](../../governance-and-automation/security-compliance-checks/inactive-workspaces.md)
+  * Administrators can resolve the vulnerability, or workspace owners can resolve the task if task delegation is enabled.
+    * [Read this article to learn how collaborators can resolve the Inactive Workspaces task.](../../point-collaborators/resolve-governance-tasks/inactive-workspaces.md)
+  * [The Inactive Workspaces report](../../reporting/analytics-reports.md#inactive-workspaces-report) has been improved to reflect the new Inactive Workspaces policy. 
+  * [For details on what will happen to the Lifecycle Management policy, please read this article.](../../governance-and-automation/lifecycle-management/deprecating-lifecycle-management.md)
+  
+* **New Actions available:**
```

Comment:
više bih išao u smjeru da su novi review typeovi dostupni umjesto ovih akcija za requestanje reviewova
tipa
new Review types available:
- Sensitivity Review
  - ovo što je i sad može ostati
- Privacy Review
  - ovo što je i sad može ostati